lib/Jsrt/ChakraCore.h

```diff
@@ -163,4 +163,346 @@ JsGetModuleHostInfo(
     _In_ JsModuleHostInfoKind moduleHostInfo,
     _Outptr_result_maybenull_ void** hostInfo);
 
+/// <summary>
+///     Called by the runtime to load the source code of the serialized script.
+///     The caller must keep the script buffer valid until the JsSerializedScriptUnloadCallback.
+///     This callback is only supported by the Win32 version of the API
+/// </summary>
+/// <param name="sourceContext">The context passed to Js[Parse|Run]SerializedScriptWithCallback</param>
+/// <param name="script">The script returned.</param>
+/// <returns>
+///     true if the operation succeeded, false otherwise.
+/// </returns>
+typedef bool (CHAKRA_CALLBACK * JsSerializedLoadScriptCallback)
+    (JsSourceContext sourceContext, JsValueRef *value,
+    JsParseScriptAttributes *parseAttributes);
+
+/// <summary>
+///     Create String Variable from C String
+/// </summary>
+/// <remarks>
+///     <para>
+///     Experimental. We may update the name or behavior until it is stable.
+///     </para>
+/// </remarks>
+/// <param name="content">Pointer to string memory.</param>
+/// <param name="length">Length of the string.</param>
+/// <param name="value">.</param>
```

Comment:
>value [](start = 17, length = 5)

Please explain the value will contain the internal representation of string which can be used in other APIs like ....